packages/utils/scaffold.ts

```diff
@@ -98,6 +99,8 @@ export default function runTransform(transformConfig: TransformConfig, action: s
 				);
 		}
 	);
+	// Add a webpack script to package.json
+	addScript("webpack", "webpack")
```

Comment:
This script is dependant on which config you have generated. Sometimes it's prod, then you'll need `--config` flag, otherwise not

Reply:
Refer - https://github.com/webpack/webpack-cli/issues/709

`webpack` script is to be anyhow generated, depending on the config, `prod` and `dev` scripts are generated.